src/Authentication/Authenticators/Session.php

```diff
@@ -710,7 +710,7 @@ private function getSessionKey(string $key)
      *
      * @param int|string|null $value
      */
-    private function setSessionKey(string $key, $value): void
+    protected function setSessionKey(string $key, $value): void
```

Comment:
It also seems the method name is not good.
This sets the key value in Session User Info.

Reply:
Updated to `setSessionUserKey`.